lib/routes/situations.js

```diff
@@ -39,4 +39,7 @@ module.exports = function(api) {
 
     api.route('/situations/:situationId/openfisca-request').get(allowShow, situations.openfiscaRequest);
 
+    api.route('/situations/:situationId/openfisca-test').get(situations.openfiscaTest);
```

Comment:
Do we really want to support `GET`? What is the use case when it seems we're taking the POSTed input to format it to YAML?